### PR TITLE
Implemented isBotReadyForExecution()

### LINKED ISSUE #161: check bot, if all activities have been configured
In order to start a bot, it should be fully configured, meaning it has a values passed in for task and application, as well as all required input fields.

---

### :question: &nbsp; Additional Questions to Answer / Consider

---

## :white_check_mark: &nbsp; Conditions of satisfaction

- [ ] Should check for a bot if all required fields have been set
- [ ] allow for easy calling of that function and return a boolean for further usage
- [ ] test the method by passing in invalid ssots, ssots that have been fully filled out, ssot with required fields missing

### PR DESCRIPTION
**Pull request by SanJSp**
Fixes #161 

## Changes
In this PR I introduce the function `isRobotExecutable `with tests for its functions `configuredRobotActivitesCorrectly `and `configuredRobotParamsCorrectly`. A Bot is Executable if all required fields have a value or are tagged as `requireUserInput `needed. Furthermore for each activity needs a `rpaTask `and `rpaApplication `specified. See the tests for further documentation.

When a robot is not ready to execute, all activities/params that do not fit the specifications are logged in the console. I also introduced a small fix so that our frontend testing suite passes.

## Checklist before merge

**Developer's responsibilities**
* [ ] **Assign** one or two developers
* [ ] **Change code** if reviewer(s) has/have requested it
* [ ] **Pull request build** has passed
* [ ] **tested locally** (in at least chrome & firefox if frontend)
* [ ] updated the **documentation**
* [ ] added **tests** where necessary

### CODE REVIEW

File: client/src/utils/BpmnToSsotParsing/BpmnToSsotParsing.js [hunk 1]
```diff
@@ -200,4 +200,4 @@ const parseBpmnToSsot = async (bpmnXml, robotId) => {
   return ssot;
 }
 
-export default parseBpmnToSsot;
+export { parseBpmnToSsot };
```
Comment (kej-jay) [inline client/src/utils/BpmnToSsotParsing/BpmnToSsotParsing.js:203]: please fix linter warning or disable for this single line

File: client/src/utils/botExecution.test.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
+/* eslint-disable no-undef */
+import {
+  configuredRobotParamsCorrectly,
+  configuredRobotActivitesCorrectly,
+} from './botExecution';
```
Comment (kej-jay) [inline client/src/utils/botExecution.test.js:5]: please rename file above and change the name in this import too

**Review by kej-jay — CHANGES_REQUESTED** [review]
minor changes, but looks good to me and seems to work

**Review by kej-jay — APPROVED** [review]

**Review by lukashueller — APPROVED** [review]
Looks very good from my point of view. I like the test suite as well.

The only thing to consider in the long run: Maybe it's better for readability to move the test objects to a separate js-file. The fact that they are at the beginning of the test.js-file I think is not so optimal.

**Review by lukashueller — COMMENTED** [review]
One more thing comes to mind: Is it really useful to output the error message that not all parameters are specified in the console? Who handles the result of the function in a form noticeable to the user? (Alert, popup or similar).

**Pull request merged**